### ISSUE TITLE
Multi page app: Interstitial closes banner & shows "Ad Not Found: 1"

### ISSUE THREAD

**Issue by royan1** [admob-plus/admob-plus#419]
I have an issue with a multi page app

Running the first index.html of the app, and loading a banner ad works fine - loading an interstitial on this page also works fine

However, loading a banner ad on index.html, then moving to a different page (page2.html), and trying to load the interstitial closes the banner ad and gives an error of "Ad Not Found: 1". Requesting the interstitial a second time then works fine.

This is happening on versions >= 1.18.0 and isn't happening on versions 1.17.0 and older, however on these versions, the interstitial doesn't show at all. I've been using admob plus with a version "0.0.0" for a while where these issues don't happen.

I'm using Cordova 10.0.0 & ios 6.2.0 in a fresh project with no other plugins, using the example code from the documentation.

I'm happy to fund up to $500 for this issue to be prioritised if that's possible?

Here is a link to a repo of the setup I used: https://github.com/royan1/testapp

Please excuse me if I've done something wrong, I'm kind of new to using git!

**Environment**

```
  AdMob Plus Environment Info:
    System:
      OS: macOS Big Sur 11.4
      CPU: (8) x64 Intel(R) Core(TM) i7-4790K CPU @ 4.00GHz
      Memory: 2.63 GB / 16.00 GB
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 15.14.0 - /usr/local/bin/node
      npm: 7.7.6 - /usr/local/bin/npm
    Managers:
      CocoaPods: 1.10.1 - /usr/local/bin/pod
    Utilities:
      Clang: 12.0.5 - /usr/bin/clang
    SDKs:
      iOS SDK:
        Platforms: iOS 14.5, DriverKit 20.4, macOS 11.3, tvOS 14.5, watchOS 7.4
      Android SDK:
        API Levels: 23, 24, 25, 26, 27, 28, 29, 30
        Build Tools: 27.0.2
        System Images: android-19 | ARM EABI v7a, android-27 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: 3.0 AI-171.4443003
      Xcode: 12.5.1/12E507 - /usr/bin/xcodebuild
    Languages:
      Java: 1.8.0_151 - /usr/bin/javac
    npmPackages:
      admob-plus-cordova: 1.21.0 => 1.21.0 
      cordova-ios: ^6.2.0 => 6.2.0 
      cordova-plugin-whitelist: ^1.3.4 => 1.3.5 
    npmGlobalPackages:
      cordova: 10.0.0
    Cordova:
      Version: 10.0.0
      Plugins:
        cordova-plugin-whitelist: ^1.3.4
        admob-plus-cordova: 1.21.0
    Ionic:
      Version: -
    AndroidManifest.xml:
      Version: -


```

**Comment by ratson** [comment]
@royan1 Would commit the test code with ads? It is missing in https://github.com/royan1/testapp/blob/master/www/js/index.js

**Comment by royan1** [comment]
> @royan1 Would commit the test code with ads? It is missing in https://github.com/royan1/testapp/blob/master/www/js/index.js

Sorry - I was developing specifically in the platforms/ios directory. I updated repo now! :)

**Comment by ratson** [comment]
@royan1 It should be fixed in `admob-plus-cordova@1.21.1`.

**Comment by royan1** [comment]
> @royan1 It should be fixed in `admob-plus-cordova@1.21.1`.

Yes this absolutely works! Thank you :) I send some funding! :)

The only other issue now is banner ads are stacking vertically if you call the banner.show method. Previously it would just refresh the banner! Maybe I am doing something wrong?
